### ISSUE TITLE
window loaded , loading events not firing when page navigates 2nd time.

### ISSUE THREAD

**Issue by Nabeelhassan** [nwjs/nw.js#5101]
*NWJS version : 0.15.4
*OS : Windows 7 64bit

*Provide a reproducible case if possible
[index.txt](https://github.com/nwjs/nw.js/files/360780/index.txt)
[inject_js_end.txt](https://github.com/nwjs/nw.js/files/360779/inject_js_end.txt)
[inject_js_start.txt](https://github.com/nwjs/nw.js/files/360782/inject_js_start.txt)
[package.txt](https://github.com/nwjs/nw.js/files/360781/package.txt)

index.txt --> index.html
inject_js_end.txt --> inject_js_end.js
inject_js_start.txt --> inject_js_start.js
package.txt --> package.json

I have provided sample files for this bug .. please look into this matter.

**Comment by Nabeelhassan** [comment]
any updates on this ???/

**Comment by Nabeelhassan** [comment]
my code seems to run on v12 .. i only refactored the window bindings as mentioned in docs .. 
if i operate the window myself all the loaded , loading events are working perfect , but if i navigate via code after the first page ha loaded no events are being fired . I am having very much trouble as i cannot upgrade to latest version

**Comment by semmel** [comment]
Those events don't  work in `nwjs 0.16.0` either after a navigation in the popup.
Got a SSL problem with the url in your example, which is easy to fix replacing the url in line 24 with https://www.google.com, and removing the timer in `inject_js_end.js` though.

Just clicked on a link on the Google page and only the log messages from the injection scripts show - the `Window` event handlers are left in the dark.

By the way the popup _zoom level_ is not working after navigation in the popup page either.
Try adding `new_win.zoomLevel = 1.5;` e.g. in line 34.

**Comment by oaleynik** [comment]
I'm observing the same effect with nw 0.16.0 on OS X. @rogerwang any thoughts on this? Are we doing something wrong? Is this some kind of breaking change or is this a bug?

**Comment by cmgustavo** [comment]
I have the same issue. Any update on this? Thanks.

**Comment by Christywl** [comment]
I can't reproduce this issue with the latest nwjs-sdk-v0.21.3 on Linux and Windows with the reporter's case.

**Comment by cmgustavo** [comment]
It 's fixed on v0.19.5.

```javascript
var gui = require('nw.gui');
gui.Window.open(oauthUrl, {
  focus: true,
  position: 'center'
}, function(new_win) {
   new_win.on('loaded', function() {
   var title = new_win.window.document.title;
   $timeout(function() {
      if (title.indexOf('XXX') == -1) {
         $scope.code = title;
         // If title is "XXX", close the new window
         new_win.close();
      }
   }, 100);
});
```

I think that this issue could be closed. Thanks!

**Comment by semmel** [comment]
The error is in `v. 0.21.3` resolved. 👍 
The events `loaded`, `document-end` and `closed` are all emitted as expected.
Thank you for fixing this! 🍰
